src/main/java/org/jabref/logic/shared/PostgreSQLProcessor.java

```diff
@@ -89,23 +83,16 @@ String escape(String expression) {
     @Override
     public void startNotificationListener(DBMSSynchronizer dbmsSynchronizer) {
         // Disable cleanup output of ThreadedHousekeeper
-        Logger.getLogger(ThreadedHousekeeper.class.getName()).setLevel(Level.SEVERE);
-
-        this.listener = new PostgresSQLNotificationListener(dbmsSynchronizer);
+        //Logger.getLogger(ThreadedHousekeeper.class.getName()).setLevel(Level.SEVERE);
 
-        PGDataSource dataSource = new PGDataSource();
-        dataSource.setHost(connectionProperties.getHost());
-        dataSource.setPort(connectionProperties.getPort());
-        dataSource.setDatabase(connectionProperties.getDatabase());
-        dataSource.setUser(connectionProperties.getUser());
-        dataSource.setPassword(connectionProperties.getPassword());
 
         try {
-            pgConnection = (PGConnection) dataSource.getConnection();
-            pgConnection.createStatement().execute("LISTEN jabrefLiveUpdate");
+            connection.createStatement().execute("LISTEN jabrefLiveUpdate");
             // Do not use `new PostgresSQLNotificationListener(...)` as the object has to exist continuously!
             // Otherwise the listener is going to be deleted by GC.
-            pgConnection.addNotificationListener(listener);
+            PGConnection pgConnection = connection.unwrap(PGConnection.class);
```

Comment:
Please remove the above comment, which contradicts the new code.

Has someone tested the new implementation?

Reply:
I can try to test it this evening. I think I have a pg sql db somewhere lying around